app/models/pame_evaluation.rb

```diff
@@ -1,7 +1,350 @@
+require 'csv'
+require 'wcmc_components'
+
 class PameEvaluation < ApplicationRecord
-  belongs_to :protected_area
+  include WcmcComponents::Loadable
+
+  belongs_to :protected_area, optional: true
   belongs_to :pame_source
   has_and_belongs_to_many :countries
 
-  validates :methodology, :year, :metadata_id, :url, presence: true
+  validates :methodology, :year, :metadata_id, presence: true
+
+  # config for loadable mixin
+  ignore_column :designation
+
+  import_by protected_area: :wdpa_id
+  import_by country: :iso_3
```

Comment:
Running the importer again I found it creates countries objects and without iso_3 (which by the way makes line 197 of this file to fail - adding `.compact` would fix it).

I'm not familiar with the Loadable gem but I wonder if this import_by line should be removed. What do you think? @benkt2

Reply:
@benkt2 To add more details. I tried importing with and without this line but for me it seems to link all evaluations to the same WDPAID and Country. Removing this line seems at least to prevent generating more countries.
Does this happen for you as well?

Reply:
Yep, needed to bump wcmc_components to 0.0.5 and change this to 
import_by countries: :iso_3
Have added checks to tests to make sure evaluations get hooked up to existing country and don't create a spurious new one